sqladmin/models.py

```diff
@@ -1199,7 +1199,7 @@ def count_query(self, request: Request) -> Select:
         By default it will select all objects without any filters.
         """
 
-        return select(func.count(self.pk_columns[0]))
+        return select(func.count(self.pk_columns[0])).select_from(self.model)
```

Comment:
Can you please add a very simple test so that we make sure this works as expected?